Description: Quick fixes for removing build paths from documentation
 Two examples escape the new "reproduciblePaths" function due to line
 wrapping.  To fix this, we:
 * Remove "peek myModel", as peeking gives no extra information, and the path
   to the source file was being wrapped.
 * Space out the definition of the (monomialIdeal, NormalToricVariety)
   method so that when "code" is called in an example, we get a 4-row net
   that won't be wrapped.
 We also increase the default printWidth for examples from 77 to 100,
 which decreases the chances of this line wrap occurring.
Origin: https://github.com/Macaulay2/M2/pull/1381
Last-Update: 2020-07-08

--- a/M2/Macaulay2/packages/RandomMonomialIdeals.m2
+++ b/M2/Macaulay2/packages/RandomMonomialIdeals.m2
@@ -1803,7 +1803,6 @@
       The key Generate points to the function that is used to generate random elements according to the given model. 
     Example
       myModel = ER(2,2,0.5)
-      peek myModel
   SeeAlso
     Model
 ///
--- a/M2/Macaulay2/packages/NormalToricVarieties/Sheaves.m2
+++ b/M2/Macaulay2/packages/NormalToricVarieties/Sheaves.m2
@@ -30,8 +30,8 @@
 	    )
 	)
     );
-monomialIdeal NormalToricVariety := MonomialIdeal => X -> monomialIdeal ideal X
-
+monomialIdeal NormalToricVariety := MonomialIdeal => X ->
+    monomialIdeal ideal X
 
 ------------------------------------------------------------------------------
 -- sheaves
--- a/M2/Macaulay2/m2/run.m2
+++ b/M2/Macaulay2/m2/run.m2
@@ -110,7 +110,7 @@
      cmd = cmd | readmode(ArgNotify,      "--notify");
      cmd = cmd | readmode(ArgSilent,      "--silent");
      cmd = cmd | readmode(ArgStop,        "--stop");
-     cmd = cmd | readmode(ArgPrintWidth,  "--print-width 77");
+     cmd = cmd | readmode(ArgPrintWidth,  "--print-width 100");
      cmd = cmd | concatenate apply(srcdirs, d -> (" --srcdir",format d));
      needsline := concatenate(" -e 'needsPackage(\"",pkgname,"\", Reload => true, FileName => \"",pkg#"source file","\")'");
      cmd = cmd | if pkgname != "Macaulay2Doc" then needsline else "";
